guide/interactions/slash-commands.md

```diff
@@ -154,9 +154,10 @@ const data = new SlashCommandBuilder()
 		option.setName('category')
 			.setDescription('The gif category')
 			.setRequired(true)
-			.addChoice('Funny', 'gif_funny')
-			.addChoice('Meme', 'gif_meme')
-			.addChoice('Movie', 'gif_movie'));
+			.addChoices(
+				{ name: 'Funny', value: 'gif_funny' },
+				{ name: 'Meme', value: 'gif_meme' },
+				{ name: 'Movie', value: 'gif_movie' }));
```

Comment:
There is an extra `)`, also you probably should break into a new line.

Reply:
Thats closing the addStringOption() my friend, also due to readability and formatting I disagree, it just looks weird in a new line.

Reply:
either fix or skip - preferably fix

Reply:
> either fix or skip - preferably fix

whats there to fix tho

Reply:
The linter warnings